integration/src/test/java/org/mule/test/spring/LookupNullObjectTestCase.java

```diff
@@ -31,26 +25,18 @@
 @Story(NULL_OBJECTS_IN_SPRING5_REGISTRY)
 public class LookupNullObjectTestCase extends AbstractIntegrationTestCase {
 
-  // TODO W-14338813
-  @BeforeClass
-  public static void ignoreJava17() {
-    assumeTrue(isJavaVersionAtMost(JAVA_11));
-  }
-
   @Override
   protected String[] getConfigFiles() {
     return new String[] {};
   }
 
   @Test
-  public void noTransactionManagerThroughMuleContext() {
-    TransactionManager txManager = muleContext.getTransactionManager();
-    assertThat(txManager, is(nullValue()));
+  public void notFoundThroughRegistryLookup() {
```

Comment:
This is kind of "assert 1 == 1". I understand that it could be testing something like "it doesn't throw any exception when the lookup should return null". If that's the case, please add a comment so it works as a regression test for such situations